wordpress-mobile/gutenberg-mobile
# Register VideoPress block behind a DEV flag

With this PR, the VideoPress block has been registered behind a DEV flag. The block itself is not complete or fully functional, but is intended to serve as a useful placeholder that to be iterated on.

## Related PRs

* `Gutenberg`: https://github.com/WordPress/gutenberg/pull/47836
* `Jetpack`: https://github.com/Automattic/jetpack/pull/28812
* `iOS`: https://github.com/wordpress-mobile/WordPress-iOS/pull/20100
* `Android`: https://github.com/wordpress-mobile/WordPress-Android/pull/17928

## To Test

Run the editor locally with both Android and iOS, then go through the following checks:

- Verify that the VideoPress block can be inserted into a post via the block inserter, the block inserter search, and the slash inserter. 
- Verify that the block is only visible when running via your local development environment, it shouldn't be visible with the PR's associated installable build.
- Verify that all automated tests pass and that there are no obvious errors associated with the addition of the block.

<hr /> 

PR submission checklist:
